internal/core/converter/tensorflow/tensorflow.go

```diff
@@ -26,31 +26,35 @@ type SessionOptions = tf.SessionOptions
 type Operation = tf.Operation
 
 type TF interface {
-	GetVector(feeds []Feed, fetches []Fetch, targets ...*Operation) (values [][][]float64, err error)
+	GetVector(inputs ...string) ([]float64, error)
+	GetValue(inputs ...string) (interface{}, error)
+	GetValues(inputs ...string) (values []interface{}, err error)
 	Close() error
 }
 
 type tensorflow struct {
 	exportDir     string
 	tags          []string
+	feeds         []OutputSpec
+	fetches       []OutputSpec
 	operations    []*Operation
 	sessionTarget string
 	sessionConfig []byte
 	options       *SessionOptions
 	graph         *tf.Graph
 	session       *tf.Session
+	ndim          int8
 }
 
-type Feed struct {
-	InputBytes    []byte
-	OperationName string
-	OutputIndex   int
+type OutputSpec struct {
+	operationName string
+	outputIndex   int
 }
 
-type Fetch struct {
-	OperationName string
-	OutputIndex   int
-}
+const (
+	TWO_DIM   int8 = 2
+	THREE_DIM int8 = 3
```

Comment:
In golang const value we shouldn't use snake case please use camel case, and I recommend below
```suggestion
	NoDim uint8 = iota
	OneDim
	TwoDim
	ThreeDim
```